packages/components/src/internal/components/editable/models.ts

```diff
@@ -301,8 +301,9 @@ export class EditorModel
      * Formats the values for an entire row into a Map<string, any>
      * @param rowIdx
      * @param useRawValues If false we format the raw values
+     * @param includeSampleDisplayVal If true we want to include the sample lookup display values in the row data
      */
-    getRowValue(rowIdx: number, useRawValues = true): Map<string, any> {
+    getRowValue(rowIdx: number, useRawValues = true, includeSampleDisplayVal = false): Map<string, any> {
```

Comment:
Somewhat unfortunate that this is sample-specific. If assay results start being associated with sources (under consideration), we'll possibly need something more general here.

Reply:
yeah....agreed. I think we could generalize this so that it was more of an "includeDisplayVal" and leave it up to the caller to set that prop accordingly. I'll look into that here.

Reply:
I change this to be a callback function so that the caller can decide which columns they want to include display values for. Can you re-review?